radix-engine-tests/tests/pool_two_resource.rs

```diff
@@ -899,6 +900,56 @@ pub fn contribute_fails_without_proper_authority_present() {
     receipt.expect_specific_failure(is_auth_error)
 }
 
+#[test]
+fn contribution_of_large_values_should_not_cause_panic() {
```

Comment:
This should actually succeed and not fail. 

Contributing two buckets of max mint amount should mean that the mint the following amount of pool units:

```math
mint\_amount = \sqrt{2^{125}} \times \sqrt{2^{125}} = 2^{125}
```

So it should actually work. I will refactor this test so we're contributing _more_ than the max mint amount.